### PR TITLE
Fixed: Ordered lists appear as unordered lists #169

### PR DESCRIPTION
**Pull request by NaincyKumariKnoldus**
Hey @elitan I have tried to fix this. Please review it now.
Thanks!

#169

### CODE REVIEW

File: docs/content/docs/platform/serverless-functions/index.mdx [hunk 1]
```diff
@@ -4,7 +4,7 @@ title: 'Creating functions'
 
 Nhost gives you the option to run serverless functions on the following runtimes:
 
-- `Node.js 14` (Both JavaScript and Typescript)
+1. `Node.js 14` (Both JavaScript and Typescript)
```
Comment (elitan) [inline docs/content/docs/platform/serverless-functions/index.mdx:7]: The problem is still occurring.

`ol` tags should render an ordered list.

[](https://user-images.githubusercontent.com/331818/153878821-6c31ee31-e077-40ee-84e2-c446f097cd37.png)

**Review by elitan — CHANGES_REQUESTED** [review]

File: docs/content/docs/platform/serverless-functions/index.mdx (hunk 1 above)
Comment (szilarddoro) [reply docs/content/docs/platform/serverless-functions/index.mdx:7]: Thanks for your contribution @NaincyKumariKnoldus! Could you please take a look into the styles affecting how a `li` element is being rendered by the application? I think we might going to find something there, that's causing this issue.

**Comment by szilarddoro** [comment]
Thanks for your contribution @NaincyKumariKnoldus! Could you please take a look into the styles affecting how a `li` element is being rendered by the application? I think we might going to find something there, that's causing this issue.

**Comment by szilarddoro** [comment]
Closing in favor of #227.

**Pull request closed without merging**